source/jobs/index.md

```diff
@@ -2,32 +2,32 @@
 layout: formatted-text
 title: Join the team
 ---
-<p class="intro-copy">There’ll soon be more seats around Convivio’s big table. It’s been a busy summer already and we’re planning to expand our team.</p>
+<p class="intro-copy">We are not currently hiring for any positions.</p>
 
-We prefer building relationships to the traditional ‘CV out of the blue’ approach — so we’d like to start hearing from people now who’d be interested in joining us in the coming months.
+We're always looking to hear from people who would be interested in working with us in the future. Drop us a line at <a href="mailto:jobs@weareconvivio.com">jobs@weareconvivio.com</a>
 
-If you’re looking for a new challenge in digital, then Convivio could be the perfect next move for your career.
+## What it's like to work at Convivio
 
-## What are we like?
+At Convivio we're not remote, [we're free-range.](https://blog.weareconvivio.com/free-range-working-an-introduction-27eb178db97c) We don't have a central office and we all work from home. We encourage flexible working. [Read more about working from home at Convivio.](/jobs/working-from-home)
```

Comment:
One tiny last query. Sorry!
Here the whole of the last sentence is a link to the working from home page. I have this weird, pathological almost, aversion to sentences as links that start with 'Read more about … ', probably totally unjustified and unjustifiable. Two sentences before this one though, the last few words say 'we all work from home' and I was just wondering if we need second a specific sentence, or somehow the two could be combined? (Is this being too picky? Sorry!)

Reply:
I think that linking using a full sentence is better for screen reader users. It's useful to give complete context about where that link is going to in isolation of the words around it. Also it feels a bit more like a persuasive call to action? I'm not sure if that's what we're aiming for though.

Reply:
Ok, cool. I'm happy from here, then. Thanks for all the updates.